packages/config/index.js

```diff
@@ -308,6 +401,16 @@ function getLevelByConfigs(pathToLevel, options, allConfigs, root) {
     return Object.keys(levelOpts).length ? levelOpts : undefined;
 }
 
+function extendConfigsPathByLayer(configs) {
+    configs.forEach(config => {
+        config.levels && config.levels.forEach(level => {
+            level.path || (level.path = level.layer + '.blocks');
```

Comment:
I think levels naming scheme must be configurable.

Reply:
This will be done soon in another PR, have patience! ;-)

Reply:
Done.

Reply:
And where is it? Some call of `naming-...` may be

Reply:
→ https://github.com/bem/bem-sdk/issues/248